src/App.css

```diff
@@ -5,65 +5,7 @@
 }
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/235873569-e6c60b0c-82f8-4300-8672-f825adbf8e2d.png)

- [ ] It would be nice if fix the links in the Table of contents which you can see in the image above to make your readme descriptive and professional 😊